packages/lib-utils/package.json

```diff
@@ -24,6 +24,8 @@
   },
   "peerDependencies": {
     "@openshift/dynamic-plugin-sdk": "^1.0.0",
+    "@patternfly/react-core": "^4.198.19",
+    "@patternfly/react-table": "^4.67.19",
```

Comment:
@fhlavac let's leave this as is for now. I'll need to think on this a bit more. Technically I believe we can have multiple different versions of a patternfly component running on a page - therefore we may not need to enforce the table component as a peer dependency.